pyriemann/utils/tangentspace.py

```diff
@@ -1,88 +1,86 @@
-""" Tangent Space """
+""" Tangent space for SPD matrices. """
+
 import numpy as np
 
 from .base import sqrtm, invsqrtm, logm, expm
 from .mean import mean_covariance
 
 
-def tangent_space(covmats, Cref):
+def tangent_space(X, Cref):
     """Project a set of SPD matrices in the tangent space.
 
     Project a set of SPD matrices in the tangent space, according to the
     reference matrix Cref.
 
     Parameters
     ----------
-    covmats : ndarray, shape (n_matrices, n_channels, n_channels)
+    X : ndarray, shape (n_matrices, n_channels, n_channels)
         Set of SPD matrices.
     Cref : ndarray, shape (n_channels, n_channels)
         The reference SPD matrix.
 
     Returns
     -------
-    T : ndarray, shape (n_matrices, n_channels * (n_channels + 1) / 2)
+    X_new : ndarray, shape (n_matrices, n_channels * (n_channels + 1) / 2)
```

Comment:
I would revert this. This technically breaks public API if someone passed named parameters. Keep it T.